scripts/tests.e2e.sh

```diff
@@ -25,6 +25,40 @@ TEMP_PATH=/tmp
 mkdir $TEMP_PATH/camino-node-1 -p
 cp -r $CAMINO_NODE_PATH $TEMP_PATH/camino-node-1/camino-node
 
+DEFAULT_SUBNET_EVM_VERSION=0.5.1
+SUBNET_EVM_VERSION=$DEFAULT_SUBNET_EVM_VERSION
+
+echo "Running e2e tests with:"
+echo VERSION_1: ${VERSION_1}
+echo VERSION_2: ${VERSION_2}
```

Comment:
was these 2 intentionally re-introduced?

Reply:
No, thanks for the catch